### PR TITLE
✨ added new annofuse wf to main rnaseq wf

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

- Updates annoFuse tools to 0.90.0
- Changes num_col default in annoFuse from 25 to 24
- Adds annoFuse workflow to main RNAseq workflow
- Updates readme with changes to main workflow and annoFuse version
- Updates image for RNAseq workflow

Closes https://github.com/d3b-center/bixu-tracker/issues/813

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Workflow passes CWLtool validation
- [x] Cavatica https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/71129222-4134-4fde-b02f-0af80e23d07f/#

**Test Configuration**:
* Environment: cwltool 3.0.20200807132242
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +126,7 @@ outputs:
   RNASeQC_Metrics: {type: File, outputSource: rna_seqc/Metrics}
   RNASeQC_counts: {type: File, outputSource: supplemental/RNASeQC_counts} # contains gene tpm, gene read, and exon counts
   kallisto_Abundance: {type: File, outputSource: kallisto/abundance_out}
+  annofuse_filtered_fusions_tsv: { type: 'File?', outputSource: annoFuse_filter/filtered_fusions_tsv, doc: "Filtred output of formatted and annotated Star Fusion and arriba results" }
```
Comment (migbro) [inline README.md:129]: typo on `Filtred`

File: README.md [hunk 2]
```diff
@@ -27,6 +28,8 @@ This workflow is the current production workflow, equivalent to this [Cavatica p
 [STAR-Fusion v1.5.0](https://doi:10.1101/120295) Fusion detection for `STAR` chimeric reads.
 ### Arriba
 [Arriba v1.1.0](https://github.com/suhrig/arriba/) Fusion caller that uses `STAR` aligned reads and chimeric reads output.
+### AnnoFuse
```
Comment (migbro) [inline README.md:31]: `annoFuse`
It's all about the brand bruh.

File: workflow/kfdrc-rnaseq-workflow.cwl [hunk 3]
```diff
@@ -175,6 +180,10 @@ inputs:
       \ for Kallisto only if single end input."}
   kallisto_std_dev: {type: 'long?', doc: "Optional input. Standard Deviation of the\
       \ average fragment length for Kallisto only needed if single end input."}
+  annofuse_genome_untar_path: {type: 'string?', doc: "This is what the path will be\
+      \ when genome_tar is unpackaged", default: "GRCh38_v27_CTAT_lib_Feb092018/ctat_genome_lib_build_dir"}
+  annofuse_col_num: {type: 'int?', doc: "column number in file of fusion name."}
```
Comment (migbro) [inline workflow/kfdrc-rnaseq-workflow.cwl:185]: maybe set the default here...unless the wf takes care of it later on

**Review by migbro — CHANGES_REQUESTED** [review]
Some minor comments. Also, probably an old oversight, but `pipeline flowchart` is out of date now, and probably should be moved up in the readme, since it's the whole wf, but then annoying stripped from the doc info since SBG will make it for you on preview page.

File: workflow/kfdrc-rnaseq-workflow.cwl (hunk 3 above)
Comment (dmiller15) [reply workflow/kfdrc-rnaseq-workflow.cwl:185]: taken care of at the tool level.

File: README.md (hunk 2 above)
Comment (dmiller15) [reply README.md]: Apple would never.

File: README.md (hunk 1 above)
Comment (dmiller15) [reply README.md]: fixed

**Review by migbro — APPROVED** [review]
Looks great, thanks!

**Review by sickler-alex — APPROVED** [review]

**Pull request merged**